src/convertToObject.js

```diff
@@ -12,7 +12,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const sourceArray = sourceString.split(';').map(el => {
```

Comment:
it is important to name vars self explained in production

```suggestion
  const sourceArray = sourceString.split(';').map(source => {
```